tests/client/button-click.test.js

```diff
@@ -59,10 +59,8 @@ describe("Scenarios: Clicking widgets", () => {
       ]);
     });
     describe("When clicked", () => {
-      test("Then it does not throw an error", () => {
-        expect(async () => {
-          await testButton.click();
-        }).not.toThrow();
+      test("Then it does not throw an error", async () => {
```

Comment:
This `async => await` construct isn't necessary. I just tried and it can be written like this:

`test("Then it does not throw an error", () => expect(testButton.click()).resolves.toBeUndefined());`

BUT: It _is_ necessary in cases where you do other stuff after the `await` so maybe it looks more consistent to leave it like this.